.github/workflows/ci.yml

```diff
@@ -18,6 +18,8 @@ env:
   rust_clippy: '1.81'
   # When updating this, also update relevant docs
   rust_min: '1.81'
+  CRT_CI_ROLE: ${{ secrets.CRT_CI_ROLE_ARN }}
+  AWS_DEFAULT_REGION: us-west-2
```

Comment:
Does the Rust SDK use AWS_REGION or AWS_DEFAULT_REGION? I think it should be AWS_REGION 

https://stackoverflow.com/questions/59961939/what-is-the-difference-between-aws-default-region-and-aws-region-system-variable

https://docs.aws.amazon.com/sdkref/latest/guide/feature-region.html